src/app/shared/autocomplete/autocomplete.component.ts

```diff
@@ -0,0 +1,86 @@
+import {Component, OnDestroy, OnInit} from '@angular/core';
+import {FormBuilder, FormGroup, Validators} from '@angular/forms';
+import {Subscription} from 'rxjs';
+import * as citiesJsonData from '../constants/cities.constant.json';
+
+export interface City {
```

Comment:
Move this interface to a new separate file

Reply:
Moved